src/components/Sidebar/Sidebar.scss

```diff
@@ -1,2 +1,29 @@
 //@import '../../styles/variables';
 //@import '../../styles/mixins';
+
+.sidebar {
+  position: fixed;
+  width: 280px;
+  max-width: 70%;
+  height: 100%;
+  left: 0;
+  top: 0;
+  padding: 20px;
+  background-color: white;
+  z-index: 200; 
+  box-shadow: 0 10px 30px 0 rgba(0, 0, 0, 0.1);
+}
+
+@media (min-width: 900px) {
+  .sidebar {
+    display: none;
+  }
+}
+
+.open {
```

Comment:
Not good. Use class variations:
```
.sidebar {
  /* your styles */
  
  &.open {
}}
```

otherwise .open will apply to the whole app. Then should be used in some common.scss file if that is this case.